### PR TITLE
Convert to one sentence per line

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR converts the "Content Management Guide" guide to follow the one sentence per line idea.
See #326

### CODE REVIEW

File: guides/common/modules/proc_importing-a-subscription-manifest-into-satellite-server.adoc [hunk 1]
```diff
@@ -19,11 +19,8 @@ endif::[]
 .Procedure
 
 . In the {Project} web UI, ensure the context is set to the organization you want to use.
-
 . Navigate to *Content* > *Subscriptions* and click *Manage Manifest*.
-
 . In the Manage Manifest window, click *Browse*.
-
```
Comment (spetrosi) [inline guides/common/modules/proc_importing-a-subscription-manifest-into-satellite-server.adoc:26]: I do not see these newlines between steps as an issue. Especially in the procedures where some steps have code blocks in them. Removing them is not an issue either so I will merge as is.

File: guides/doc-Content_Management_Guide/topics/Managing_Content_Views.adoc [hunk 2]
```diff
@@ -23,22 +29,24 @@ This ensures systems are designated to a specific environment but receive update
...
 
- . Create a Content View.
- . Add the repository and the Puppet modules that you want to the Content View.
- . Optionally, create one or more filters to refine the content of the Content View.
- . Optionally, resolve any package dependencies for a Content View.
- . Publish the Content View.
- . Optionally, promote the Content View to another environment.
- . Attach the content host to the Content View.
-
+. Create a Content View.
```
Comment (spetrosi) [inline guides/doc-Content_Management_Guide/topics/Managing_Content_Views.adoc:32]: lol this was weird, thank you for fixing.

File: guides/doc-Content_Management_Guide/topics/Managing_Content_Views.adoc [hunk 3]
```diff
@@ -272,10 +292,10 @@ If you want to develop an application that supports different database servers.
 
 Example of four separate Content Views:
 
-  * Red Hat Enterprise Linux (Operating System)
-  * PostgreSQL (Database)
-  * MariaDB (Database)
-  * _example_software_ (Application)
+* Red Hat Enterprise Linux (Operating System)
```
Comment (spetrosi) [inline guides/doc-Content_Management_Guide/topics/Managing_Content_Views.adoc:295]: :/

**Review by spetrosi — APPROVED** [review]
Thanks for your effort @maximiliankolb

**Comment by maximiliankolb** [comment]
Thanks for merging. One PR to go. :)

**Pull request merged**